src/PaymentGateways/PayPalStandard/Migrations/RemovePayPalIPNVerificationSetting.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Give\PaymentGateways\PayPalStandard\Migrations;
+
+use Give\Framework\Migrations\Contracts\Migration;
+
+/**
+ * @unreleased
+ */
+class RemovePayPalIPNVerificationSetting extends Migration {
```

Comment:
I'm on the fence as to whether this is really worth removing. I get that we're otherwise leaving data in `wp_options`; it just feels like a lot for removing a single line. It's probably fine. 🤔